include/envoy/ssl/connection.h

```diff
@@ -88,6 +88,12 @@ class Connection {
    *         considered valid after. Returns empty absl::optional if there is no peer certificate.
    **/
   virtual absl::optional<SystemTime> expirationPeerCertificate() const PURE;
+
+  /**
+   * @return std::string the URL-encoded PEM-encoded representation of the peer certificate chain.
+   *         Returns "" if there is no peer certificate or encoding fails.
+   **/
+  virtual const std::string& urlEncodedPemEncodedPeerCertificateChain() const PURE;
```

Comment:
Nit: you might consider moving this right after `urlEncodedPemEncodedPeerCertificate()`, so that similar functions are grouped together (same in tests).

Reply:
Addressed in 876e4a5.